src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -82,6 +83,42 @@ public FeedbackSessionAttributes getFeedbackSession(String courseId, String feed
         return feedbackSession;
     }
 
+    /**
+     * Gets a list of feedback sessions within the given time range.
+     */
+    public List<FeedbackSessionAttributes> getAllFeedbackSeesionWithinTimeRange(Instant rangeStart, Instant rangeEnd) {
+
+        List<FeedbackSessionAttributes> lst = new LinkedList<>();
+
+        List<FeedbackSession> startEntities = load()
+                .filter("startTime >=", rangeStart)
+                .filter("startTime <", rangeEnd)
+                .list();
+        List<FeedbackSession> endEntities = load()
+                .filter("endTime >=", rangeStart)
+                .filter("endTime <", rangeEnd)
+                .list();
+
+        List<FeedbackSession> endTimeEntities = new ArrayList<>(endEntities);
```

Comment:
Are `endTimeEntities` and `startTimeEntities` necessary? They don't seem to be used

Reply:
It is actually a bug, I should use endTimeEntities below. thanks for pointing out.